src/scripts/exclusion_term_expansion/exclusion_term_expansion.py

```diff
@@ -0,0 +1,316 @@
+"""Takes in the full ontology and the exclusions tsv and extracts a simple list of terms from it.
+Outputs two files:
+1. a simple list of terms (config/ONTO_NAME_term_exclusions.txt)
+2. a simple two column file with the term_id and the exclusion reason
+
+# Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/22
+
+TODO's
+  - todo: Do we need to worry about edge case where exclusion table is empty? (e.g. we include such a table for all
+     ontologies, but there exists 1 where we don't exclude anything.
+  - todo: later: see below: #x1: regex results: would be good to filter out by IRI or prefix that is not from source
+     ontology, but how to determine the appropriate prefix just based on the 2 file params, but could have edge cases.
+  - todo: later: see below: #x2
+  - todo: later: see below: #x3
+  - todo: later: see below: #x4: Switch from SPARQL to OAK.
+  - todo: later: When constructing exclusion TSV: bug: ICD10CM:C7A-C7A reported as term, but should be ICD10CM:C7A
+  - todo: later: QA: possible conflicts in icd10cm_exclusions.tsv: Add a check to raise an error in event of a parent
```

Comment:
@matentzn W/ the holiday this week and a sick day, I'm a bit low on time. I think this edge case is a low probability, so I have just added a todo comment for now. I don't think this is something I can do in like ~10 minutes. Probably would take me more than 30. So I think this is a low priority right now w/ the other stuff on my plate.

Reply:
ok make an issue instead. Good prioritsation.